### PR TITLE
use local npm configuration

### PR DESCRIPTION
**Pull request by tsatse**
gets registry url from local npm configuration
and use locally linked modules, if present, instead of installing them

### CODE REVIEW

File: bundler/registry.js [hunk 1]
```diff
@@ -35,13 +42,23 @@ registry.metadata = function metadata(module, cb) {
...
 registry.resolve = function resolve(module, version, cb) {
-  registry.versions(module, version, function (err, vs) {
-    if (err) return cb(err);
-    cb(null, vs[0]);
+  console.log('*** registry.resolve');
+
+  npm.load({}, function() {
+    npmRegistry = npm.config.get('registry');
+    
+    console.log('npm registry is ', npmRegistry);
```
Comment (jfhbrook) [inline bundler/registry.js]: Isn't there a logging module in place we should be using?

**Comment by jfhbrook** [comment]
Why would I want this? Not rejecting this out-of-hand, but for the added complexity of an already complex project I need a plausibility argument.

**Comment by tsatse** [comment]
I admit that it adds complexity and I issued this just in case you find it useful but feel free to reject it.
- I found it handy to use your project with my own npm registry
- detecting locally linked modules makes it easier for me to do tests from my dev box

File: bundler/registry.js (hunk 1 above)
Comment (tsatse) [inline bundler/registry.js]: true, those were not meant to remain here

**Comment by tsatse** [comment]
Ah forget it, it's more a responsability of the npm repo to do that

**Comment by akenn** [comment]
I was just about to open a PR for this. How about just allowing the registry url to be passed in as an environment variable e.g. `REGISTRY=http://registry.mydomain.com npm start`? I don't think that would add complexity.

**Comment by jfhbrook** [comment]
I merged akenn's PR, so I think it's best if I shut this one down. I think it addresses similar concerns.

Thanks for the suggestion!

**Pull request closed without merging**